packages/expo-modules-core/android/proguard-rules.pro

```diff
@@ -33,3 +33,7 @@
 -keep class * implements expo.modules.kotlin.views.ComposeProps {
   *;
 }
+
+-keepnames class * implements expo.modules.kotlin.views.ExpoView {
```

Comment:
We have similar rule above, can we merge it?

Reply:
ohno, saw the approve and landed this 😢 
We can merge it if you prefer, though imo the syntax is a bit cleaner as separate entries.
`keepnames` should also work for composeprops though, so we can change that rule.